packages/runner/src/suite.ts

```diff
@@ -495,6 +503,7 @@ function createSuiteCollector(
 
     allChildren.forEach((task) => {
       task.file = file
+      task.fullName = createTaskName([file.fullName, task.fullName])
```

Comment:
Not entirely sure about this. `file` seems to be "injected" at a later stage, so to include the file path I had to change all children's `fullName`. Is there a better way of doing this?